pyston/serializer.py

```diff
@@ -409,7 +420,11 @@ def _get_allowed_fieldset_from_resource(self, model_resource, obj, via, has_get_
             return model_resource.get_fields(obj)
 
     def _get_fieldset(self, obj, extended_fieldset, requested_fieldset, exclude_fields, via, detailed,
-                      direct_serialization):
+                      direct_serialization, serialized_objects):
+
+        if self._get_obj_serialization_name(obj) in serialized_objects:
+            return rfs((get_last_parent_pk_field_name(obj)))
```

Comment:
pokud už byl v minulosti serializován (o nějaké to patro výše va stromu), vypíše se jen pole pk